### PR TITLE
Darlp networks caption settings 1203376941237109

### PR DESCRIPTION
**Pull request by darlp5589**
Hi @triveni-codev @chriscodev @pauljustinhernandez 

Kindly check this PR. This is only for Default Service Caption Settings tab under Services only for ATSC Terrestrial type

ASANA: https://app.asana.com/0/1201943683850147/1203376941237109

To test:

-should have the same set and order of languages from Java App
-should be able to retrieve service caption settings from Java App and add/update them
-should be able to add one or more service caption settings
-should be able to update one or more service caption settings
-cancellation should be working fine including all upper modals: networks, channels, services and service caption settings modal
-test Big Commit for Adding (Updating is still under development)

Don't test Big Commit for Updating yet.

### CODE REVIEW

**Comment by chriscodev** [comment]
hi @darlp5589 ,

Here are my Feed back and  few concerns:
============================================
-should have the same set and order of languages from Java App [passed]
-should be able to retrieve service caption settings from Java App and add/update them [passed] 
-should be able to add one or more service caption settings [passed]
-should be able to update one or more service caption settings [passed]
-cancellation should be working fine including all upper modals: networks, channels, services and service caption settings modal[passed]

here are my feedback

modals backdrop can be clicked it should not be, this will cause error for future  use,
use this sample settings for modal backdrop
$('#modalEditLLSSettings').modal({
    backdrop: 'static',
    keyboard: false,
  });


qa1
the "Default Service Caption Setting" and "Linked Scehdule Caption Settings" must have same  layout design on servicesettings same in java app

qa2
remove classname "form-material" this is using different css pattern and not using elite admin causing to have different layout on input

qa3
on input  service number and highlight and change to zero, can proceed to adding,  it should not be cause valuerange is 1-63

qa1
qa1

qa2
qa2

qa3
qa3

**Comment by darlp5589** [comment]
Hi @chriscodev ,

Here are my responses:
qa1
the "Default Service Caption Setting" and "Linked Scehdule Caption Settings" must have same layout design on servicesettings same in java app [Please disregard for now. I will be working on the Linked Schedule Caption Settings in the next ticket but this is noted.]

qa2
remove classname "form-material" this is using different css pattern and not using elite admin causing to have different layout on input [Done]

qa3
on input service number and highlight and change to zero, can proceed to adding, it should not be cause valuerange is 1-63 [Done]

**Review by chriscodev — APPROVED** [review]
LGTM

**Comment by triveni-codev** [comment]
Approved to a certain point only. 
This is the next and ready for review:
image

**Comment by triveni-codev** [comment]
Leaving a note. Approved to a certain point only and review will be continued. 
✅should have the same set and order of languages from Java App
✅should be able to retrieve service caption settings from Java App and add/update them
✅should be able to add one or more service caption settings
✅should be able to update one or more service caption settings
✅cancellation should be working fine including all upper modals: networks, channels, services and service caption settings modal
✅test Big Commit for Adding
❌Big commit for Updating. <----To be approved including the interplay with add operation


Hi @chriscodev, This is the next and ready for review:
image

**Comment by chriscodev** [comment]
hi @darlp5589 ,

here are my review  on this PR


qa2
from server
	- when i tried to update exsiting data ang go to adding atsc channel there it exist 1 data and i add new atsc service then commit it and view again the newly added data then update it the atsc service button is disabled

qa2
from server
	every time i update the atsc service and commit it when viewing,  it adds duplicate data after saving it causing to have multiple data when viewing

qa1
qa1

qa2
qa2

Cheers,
Christian

**Comment by darlp5589** [comment]
qa1
from server
- when i tried to update exsiting data ang go to adding atsc channel there it exist 1 data and i add new atsc service then commit it and view again the newly added data then update it the atsc service button is disabled
[May I know how many services you have and what's the max service being specified in the features? It might be the reason for having the disabled button. I can't replicated on my end.]

qa2
from server
every time i update the atsc service and commit it when viewing, it adds duplicate data after saving it causing to have multiple data when viewing
[Can you send me a video on this or maybe walk me through how to replicate instead? Thanks!]

**Comment by darlp5589** [comment]
Consider this PR as closed as the recent concerns were already addressed on the new PR that branches out from this. You may refer the conversation here: https://github.com/triveni-codev/GBWebApp-dev/pull/510

**Comment by triveni-codev** [comment]
PR History Tree:
[PR 513](https://github.com/triveni-codev/GBWebApp-dev/pull/513/commits)
↑
[PR 510](https://github.com/triveni-codev/GBWebApp-dev/pull/510/commits) ❌<- Obsolete & completely faced out due to anomaly where 9 files are modified and previous functionalities got removed after intentionally updating one line (Image reference):
image
↑
[PR 508](https://github.com/triveni-codev/GBWebApp-dev/pull/508/commits)


TLDR: PR 508 is part of PR 513. When PR 513 will be merged, PR 508 codes is already bundled with it.
